srml/bridge/Cargo.toml

```diff
@@ -8,9 +8,15 @@ edition = "2018"
 
 [dependencies]
 codec = { package = "parity-scale-codec", version = "1.0.0", default-features = false }
+client = { package = "substrate-client", path = "../../core/client" }
+fg = { package = "substrate-finality-grandpa", path = "../../core/finality-grandpa/" }
```

Comment:
You can't import this.

Reply:
I'm gonna wait on #3868 before getting rid of it